java/src/main/native/src/ScalarJni.cpp

```diff
@@ -345,10 +344,10 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_Scalar_makeTimestampDaysScalar(JNIEn
   CATCH_STD(env, 0);
 }
 
-JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_Scalar_makeDurationScalar(JNIEnv *env, jclass,
-                                                                           jint jdtype_id,
-                                                                           jlong value,
-                                                                           jboolean is_valid) {
+JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_Scalar_makeDurationTimeScalar(JNIEnv *env, jclass,
+                                                                      jint jdtype_id,
```

Comment:
Indentation is still off here.

Reply:
I don't know how I missed this one, as I specifically remember fixing the indentation